src/namespaces/destinations/relewise.d.ts

```diff
@@ -0,0 +1,154 @@
+import { BaseFullSchema } from "../../schemas/base-full-schema";
+
+/**
+ * Relewise destination
+ */
+export namespace Destinations.Relewise {
+
+    export namespace Schemas {
+        
+        export class Product extends BaseFullSchema<Types.ProductProperties, Types.ProductDestinationOptions> {
+        }
+        
+        export class Content extends BaseFullSchema<Types.ContentProperties, Types.ContentDestinationOptions> {
+        }
+
+        export class Brand extends BaseFullSchema<Types.BrandProperties, Types.BrandDestinationOptions> {
+        }
+
+        export class ProductCategory extends BaseFullSchema<Types.ProductCategoryProperties, Types.ProductCategoryDestinationOptions> {
+        }
+
+        export class ContentCategory extends BaseFullSchema<Types.ContentCategoryProperties, Types.ContentCategoryDestinationOptions> {
+        }
+    }
+
+    export namespace Types {
+
+        interface ProductProperties  {
+            product: {
+                displayName?: Multilingual
+                assortments?: number[]
+                listPrice?: MultiCurrency
+                salesPrice?: MultiCurrency
+                CategoryPaths?: CategoryPath[]
+                brand?: Brand
+                custom?: { [key: string]: string; } 
+                data?: { [key: string]: DataValue; } 
+            }
+            variants?: {
+                displayName?: Multilingual
+                assortments?: number[]
+                listPrice?: MultiCurrency
+                salesPrice?: MultiCurrency
+                custom?: { [key: string]: string; } 
+                specification?: { [key: string]: string; } 
+                data?: { [key: string]: DataValue; } 
+            }[]
+        }
+
+        class ProductDestinationOptions {
+            id: string
+            relewiseEntityType: 'product'
```

Comment:
Do you think we could simplify `options.{..}type` property names? maybe excluding `relewise` from them, since we users hopefully by then are aware that they are working with Relewise schema.

Reply:
I'm in an Enterspeed schema with a Relewise destination. Not sure it's 100% clear if it's an Enterspeed type or Relewise type I need to provide. Maybe it could be simplicied to `relewiseType` instead of `relewiseEntityType`?

Reply:
I think you are the best judge here